src/Fingerprint.js

```diff
@@ -71,12 +72,11 @@ $.extend( SELF.prototype, {
 	},
 
 	/**
-	 * @param {string} languageCode
 	 * @param {wikibase.datamodel.Term} label
 	 * @return {boolean}
 	 */
-	hasLabel: function( languageCode, label ) {
-		return this._labels.hasItem( languageCode, label );
+	hasLabel: function( label ) {
```

Comment:
Note that we already have `hasLabelFor` below which does exactly that. We could just remove this method but I think it is useful if you don't only want to check if a label is set but also compare it to the value passed here.